Doc/library/stdtypes.rst

```diff
@@ -265,9 +265,8 @@ which is narrower than complex.  Comparisons between numbers of mixed type use
 the same rule. [2]_ The constructors :func:`int`, :func:`float`, and
 :func:`complex` can be used to produce numbers of a specific type.
 
-All numeric types (except complex) support the following operations, sorted by
-ascending priority (all numeric operations have a higher priority than
-comparison operations):
+All numeric types (except complex) support the following operations (for priorities of 
```

Comment:
Travis indicated that there is trailing whitespace here, it needs to be removed.

```suggestion
All numeric types (except complex) support the following operations (for priorities of
```

Reply:
Thank you for reviewing my pull request. :)
It's done.